procrastinate/admin.py

```diff
@@ -0,0 +1,174 @@
+from procrastinate import connector as connector_module
+from procrastinate import sql, utils
+
+
+@utils.add_sync_api
+class Admin:
+    """
+    The Admin is used to overview and administrate procrastinate jobs.
+
+    You should never need to instanciate an Admin object as it is
+    already available as `App.admin`.
+    """
+
+    def __init__(self, connector: connector_module.BaseConnector):
+        """
+        Parameters
+        ----------
+        connector:
+            Instance of a subclass of :py:class:`procrastinate.connector.BaseConnector`,
+            typically `PostgresConnector`. It will be responsible for all communications
+            with the database. Mandatory.
+        """
+        self.connector = connector
+
+    async def list_jobs_async(
+        self, id=None, queue=None, task=None, status=None, lock=None,
+    ):
```

Comment:
Do you want to add types for the function arguments and return values?

Reply:
Yes, I think it's relevant for this class.